egs/hkust/s5b/local/hkust_data_prep.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+
+
+. path.sh
+
+if [ $# != 2 ]; then
+   echo "Usage: hkust_data_prep.sh AUDIO_PATH TEXT_PATH"
```

Comment:
change message to be in this style:
```
Usage: $0 <audio-path> <text-path>
e.g.: $0 /export/corpora/LDC43142B /export/corpora/LDC43142X
```
... obviously use the correct directories, that look plausible (maybe it's not LDC).